app/client/components/pages/singleUserPage.jsx

```diff
@@ -95,7 +96,7 @@ class SingleUserPage extends React.Component {
   }
 }
 
-export default connect(mapStateToProps, mapDispatchToProps)(SingleUserPage);
+export default connect(mapStateToProps, mapDispatchToProps)(organizationWrapper(SingleUserPage));
```

Comment:
It could potentially make more sense for the HOC to be used in the `nav.jsx` file, so we can easily see which pages it's applied to?

Reply:
yeah! I originally wanted to do this and I just tried it again for a sanity check but what happens is that the nav file uses lazy loading for the components that would need to be wrapped. the wrapper expects the component immediately and so i'm not too sure how to incorporate the HoC in the nav file without removing lazy loading

Reply:
yeah, I'm not sure. I think it's because the lazy loading doesn't immediately render a component, so  the HOC throws an error

Reply:
do you think this is blocking? I can look into a workaround before merging this PR if so

more broadly, I was going to bring up a possible discussion about reformatting the routes because I think in this hierarchy organizations should be the inner most path (ex: /org/1/accounts, /org/2/transfers) because the accounts and transfers are per org

Reply:
no, I think what you've got so far is fine 😄 

Yes, definitely open to reformatting the routes. We originally looked at doing that, and I think it would be a better way that as a query string in the long run

Reply:
https://trello.com/c/15EDvhDn/258-reformat-dashboard-routes-to-better-reflect-hierarchy